cfgov/unprocessed/css/pages/home.less

```diff
@@ -0,0 +1,106 @@
+.content_main__home {
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+
+    .respond-to-max( @bp-xs-max, {
+        padding-bottom: unit( ( @grid_gutter-width * 2 ) / @base-font-size-px, em );
+    } );
+}
+
+.content-l_col__home {
+    + .content-l_col__home {
+        .respond-to-max( @bp-med-max, {
+            margin-top: unit( ( @grid_gutter-width ) / @base-font-size-px, em );
+        } );
+
+        .respond-to-range( @bp-sm-min, @bp-med-max, {
+            &:before {
+                .a-rule-break();
+                content: '';
+                display: block;
+            }
+        } );
+    }
+
+    .respond-to-min( @bp-sm-min, {
+        .grid_column( 1, 1 );
+    } );
+
+    .respond-to-min( @bp-lg-min, {
+        .grid_column( 4 );
+    } );
+}
+
+.o-well__home {
+    .respond-to-min( @bp-xl-min, {
+        // Max-width - both gutters: required due to well spaning the full page
+        max-width: 1230px - ( 45px * 2 );
+        margin: 0 auto;
+        padding-right: unit( 45px / @base-font-size-px, em );
+        padding-left: unit( 45px / @base-font-size-px, em );
```

Comment:
I'm going to leave it because we still haven't made these global variables, but I'm adding a TODO for after launch.